tests/format/flow/optional-indexed-access/jsfmt.spec.js

```diff
@@ -0,0 +1 @@
+run_spec(__dirname, ["flow"], { trailingComma: "all" });
```

Comment:
`babel-flow` doesn't support it, correct?

Reply:
Maybe Babel 7.14.0 supports it, but we haven't updated yet

Reply:
If that's true, better update babel first.

Reply:
Babel 7.14.0 does support it (https://github.com/babel/babel/releases/tag/v7.14.0), didn't realize it had released (happened just today)

Reply:
However probably just update to 7.14,  tests will be failed. Because almost error messages thrown by babel/parser are updated.

Reply:
I'll create PR to update babel/parser

Reply:
I'm not doing anything now, I can try.

Reply:
Sorry I've created https://github.com/prettier/prettier/pull/10802

Reply:
I was busy... suddenly